### DISCUSSION TITLE
Warn user of fallback images when RCT1 not linked.

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#17927] [Feature requests] [answered]
I would like to warn users in the object selection window debug info, make a pop-up when the user adds an object to the selected objects, and make a pop-up when the user loads a game with objects requiring fallback sprites.

I am attempting to implement this feature myself, but it is proving more arduous than anticipated.

EDIT: Apparently I was just trying to do one thing too hard and I should do another thing.

**Comment by Gymnasiast** [comment, marked as answer]
Duplicate of #17806.
